### ISSUE TITLE
Create a gitter.im lobby.

### ISSUE THREAD

**Issue by grabear** [grunwaldlab/metacoder#229]
Create a gitter.im lobby for chatting, and add a gitter badge to the README file.  This is the hypothetical link you would use:

[![Join the chat at https://gitter.im/metacoder/Lobby](https://badges.gitter.im/metacoder/Lobby.svg)](https://gitter.im/metacoder/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

I would create the PR myself, but I think I would have to have access to the grunwaldlab organization or something like that.

**Comment by zachary-foster** [comment]
Good idea! I have not got it to work however, maybe a bug or something. It says something like "needs private repo access", but when I click to enable private repo access nothing happens and I cant create a channel anymore. Its odd because `metacoder` is not private.. I will look into this again later.

**Comment by grabear** [comment]
@zachary-foster,

> I'd love to help with this package more.  For instance I could set up gitter for you if you'd like, and I could start integrating ZenHub into this repository.  I'd also like to help with phyloseq if at all possible.
> 
> Just tell me what you need help with and I can devote some time to it every week! 

Cheers,
Rob Gilmore

**Comment by zachary-foster** [comment]
Thanks @grabear! I dont know anything about ZenHub. Feel free to set it up. Gitter sounds great. What permissions would you need to set those up? 

If you wanted to do some coding, there are a few things that have been on the wish list for a while that I dont see myself getting to anytime soon. How about this one: https://github.com/grunwaldlab/metacoder/issues/105

Thanks for the offer to help!

**Comment by grabear** [comment]
I'd need to be a contributor with read and write permissions (I think).  When I get back to my lab, I'll give you a better answer.

**Comment by grabear** [comment]
@zachary-foster 

So for Gitter, I would need to be added as a member of your organization, and then I would need to be added as a collaborator to the repository.  From there I should be able to properly set up the gitter lobby.

For ZenHub, you need to go to www.ZenHub.com and install their browser [extension for chrome or firefox](https://www.zenhub.com/extension) (do you use one of these?)  After being added as a collaborator, both of us will be able to work in the same ZenHub dashboard, which just modifies your GitHub page:

# ZenHub Tab

![image](https://user-images.githubusercontent.com/20953530/40557226-11fdda84-6015-11e8-909f-623c12caa615.png)

# ZenHub Dashboard

![image](https://user-images.githubusercontent.com/20953530/40557264-344e9632-6015-11e8-9616-9aadb1193d12.png)

It doesn't look great right now, but that's because it just auto-populates your current issues.  I can give you a better break down of the functionality once it's up and running.

# Contributions

> How about this one: #105

I could certainly work on a barchart.  We've done that for our projects.  I could work on correlation plots too.  I've done that to some degree with metacoder already.

**Comment by zachary-foster** [comment]
Thanks for the info @grabear! I am heading out for some camping for the next few days. When I get back I will look over all this in more detail.

**Comment by zachary-foster** [comment]
> So for Gitter, I would need to be added as a member of your organization, and then I would need to be added as a collaborator to the repository. From there I should be able to properly set up the gitter lobby.

I am going to talk to my advisor @grunwald about this when he gets back tomorrow. I was messing around with gitter and I think he might need to make it since its his organization. 

> For ZenHub...

I think I have it working. Thanks for the info!

> I could certainly work on a barchart. We've done that for our projects.

Great! Let me know if you have any questions. Thanks again! 

>  I could work on correlation plots too. I've done that to some degree with metacoder already.

What do you mean by "correlation plots"?

**Comment by grabear** [comment]
Glad you got ZenHub working.  It takes a little getting used to, but the Epics that you can create are invaluable.

> What do you mean by "correlation plots"?

@zachary-foster I created another issue to explain correlation plots #234

**Comment by grabear** [comment]
Also thanks for adding me to metacoder.  I'm excited to work on it.

**Comment by zachary-foster** [comment]
Closed due to age. If this is still relevant feel free to reopen.
